src/firebolt/model/engine.py

```diff
@@ -38,6 +38,14 @@ class EngineKey(FireboltBaseModel):
     engine_id: str
 
 
+def wait(seconds: int, timeout_time: float, error_message: str, verbose: bool) -> None:
+    time.sleep(seconds)
+    if time.time() > timeout_time:
```

Comment:
I think it should be >=

Reply:
Switching from `>` to `>=` would make a nanosecond difference